examples/executor.yml

```diff
@@ -1,6 +1,8 @@
 ---
 listen_address: ":23222"
 verbose: false
+tls_key: "certs/key.pem"
+tls_crt: "certs/cert.pem"
```

Comment:
```suggestion
# tls_crt: "certs/cert.pem"
```

Comment-out so that people running examples as-is aren't required to also create a self-signed certificate.